common/config/rush/pnpm-lock.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   '@rush-temp/tree-widget-react': 'file:projects/tree-widget-react.tgz'
```

Comment:
@calebmshafer , we need to remove you as a codeowner to the the lockfile

Reply:
Yea feel free to open a PR today so I can be removed before going on vacation next week.  Also, should probably find someone, in addition to me, who can approve all any change to the repo.